packages/core/src/odata/common/request/odata-request.ts

```diff
@@ -189,6 +180,27 @@ export class ODataRequest<RequestConfigT extends ODataRequestConfig> {
     }
   }
 
+  /**
+   * Create object containing all basic headers for the given request, including custom headers, but excluding destination related and csrf headers.
+   *
+   * @returns Key-value pairs where the key is the name of a header property and the value is the respective value
+   */
+  basicHeaders(): Record<string, any> {
+    const defaultHeaders = replaceDuplicateKeys(
```

Comment:
what is the difference of replaceDuplicate keys and using the `...` operator?

Reply:
Discussed in other PR.